### PR TITLE
Updated Okta Sign-In Widget and Angular code samples.

### PR DESCRIPTION
**Pull request by KostiantynDrozd-okta**
## Description:
- **Updated code samples for Okta Sign-In Widget and Angular** 

### Resolves:

* [OKTA-348833](https://oktainc.atlassian.net/browse/OKTA-348833)

### CODE REVIEW

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 1]
```diff
@@ -208,19 +211,9 @@ export class LoginComponent implements OnInit {
...
-          this.signIn.loginRedirect('/', { sessionToken: res.session.token });
-          // Hide the widget
-          this.widget.hide();
-        }
-      },
-      (err) => {
-        throw err;
-      }
-    );
+    this.widget.showSignInAndRedirect().catch(err => {
```
Comment (swiftone) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:214]: Wouldn't this be `.showSignInAndGetTokens()`?  Redirect should be reserved for okta-hosted situations.

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 2]
```diff
@@ -119,12 +119,12 @@ export class AppComponent implements OnInit {
   }
 
   login() {
-    this.oktaAuth.loginRedirect('/profile');
+    this.oktaAuth.signIn('/profile');
```
Comment (aarongranick-okta) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:122]: Should be:

```
this.oktaAuth.signInWithRedirect({
  originalUri: '/profile'
})
```

There's a [migrating guide](https://github.com/okta/okta-angular/blob/master/MIGRATING.md#loginredirect-is-replaced-by-signinwithredirect) that can help.

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md (hunk 1 above)
Comment (KostiantynDrozd-okta) [reply packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:214]: @swiftone 
I replaced .showSignInAndRedirect() to .showSignInAndGetTokens(), and I get and error:
`core.js:4352 ERROR TypeError: this.widget.showSignInAndGetTokens is not a function`...

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md (hunk 2 above)
Comment (KostiantynDrozd-okta) [reply packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md]: @aarongranick-okta 
Done!

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md (hunk 1 above)
Comment (KostiantynDrozd-okta) [reply packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:216]: @swiftone 
Using `.showSignInToGetTokens()` no redirection was occurred.
And with `.showSignInAndRedirect()` application have the same behavior as before.

Comment (aarongranick-okta) [reply packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:216]: I think its OK to preserve existing behavior in this case

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 3]
```diff
@@ -119,12 +119,14 @@ export class AppComponent implements OnInit {
...
-    this.oktaAuth.loginRedirect('/profile');
+    this.oktaAuth.signInWithRedirect({
+      originalUri: '/profile'
+    });
   }
 
   async logout() {
     // Terminates the session with Okta and removes current tokens.
-    await this.oktaAuth.logout();
+    await this.oktaAuth.signOut();
```
Comment (aarongranick-okta) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:129]: I believe this function will perform a full redirect to Okta and back to the application. So there may be no need for the router navigateByUrl

**Review by aarongranick-okta — APPROVED** [review]

**Pull request merged**